assets/js/components/DatabaseDetails/DatabaseDetails.stories.jsx

```diff
@@ -7,7 +7,7 @@ import {
   databaseFactory,
   hostFactory,
 } from '@lib/test-utils/factories';
-import { DATABASE_TYPE } from '@lib/model';
+import { APPLICATION_TYPE, DATABASE_TYPE } from '@lib/model';
```

Comment:
I don't think we need the `APPLICATION_TYPE` here. This should be hardcoded, as this is the Database view